pybuildkite/builds.py

```diff
@@ -75,6 +75,7 @@ def list_all(
         meta_data=None,
         branch=None,
         commit=None,
+        include_retried_jobs=False,
```

Comment:
If we decide on the comment about `query_params` it may make more sense to predefine this as `None`

Reply:
I am fine with either default, let me know which you prefer.

Reply:
Let's go with `None` for consistency =).